src/qtmozembed/declarativewebpage.h

```diff
@@ -31,6 +31,8 @@ class DeclarativeWebPage : public QOpenGLWebPage {
     Q_PROPERTY(bool userHasDraggedWhileLoading MEMBER m_userHasDraggedWhileLoading NOTIFY userHasDraggedWhileLoadingChanged FINAL)
     Q_PROPERTY(bool fullscreen READ fullscreen NOTIFY fullscreenChanged FINAL)
     Q_PROPERTY(bool forcedChrome READ forcedChrome NOTIFY forcedChromeChanged FINAL)
+    Q_PROPERTY(bool domContentLoaded READ domContentLoaded NOTIFY domContentLoadedChanged FINAL)
```

Comment:
This is a bit unrelated but handy addition when you need debug things. You can do debugging quicker from QML side.